gyro_averages.f90

```diff
@@ -248,9 +118,368 @@ subroutine init_bessel
             end do
          end do
       end if
+      if (debug) write (*, *) 'gyro_averages::init_bessel::test_gyro_average'
+!    call test_gyro_average
+
+   contains
+
+!     ! set up field that varies as x^2 = rho^2 * cos(angle)^2,
```

Comment:
Could delete this commented out subroutine?

Reply:
this subroutine contains a test that would probably be useful to have around; perhaps would be better to move to a dedicated tests module / folder...